tests/sentry/api/test_organization_missing_members.py

```diff
@@ -0,0 +1,88 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.testutils import APITestCase
+from sentry.testutils.silo import region_silo_test
+
+
+@region_silo_test(stable=True)
+class OrganizationMissingMembersTestCase(APITestCase):
+    endpoint = "sentry-api-0-organization-missing-members"
+    method = "get"
+
+    def setUp(self):
+        super().setUp()
+
+        self.create_member(
+            email="a@example.com",
+            organization=self.organization,
+        )
+        member = self.create_member(user=self.create_user(), organization=self.organization)
+        member.user_email = "b@example.com"
+        member.save()
+
+        self.commit_author = self.create_commit_author(project=self.project, email="b@example.com")
+        self.commit_author1 = self.create_commit_author(project=self.project, email="c@example.com")
```

Comment:
Could we give these better names so we know what's different about them? Like if one is not a member name them `non_member_commit_author` or whatever